src/gui/TrackContentObjectView.cpp

```diff
@@ -568,7 +578,34 @@ void TrackContentObjectView::mousePressEvent( QMouseEvent * me )
 	setInitialOffsets();
 	if( !fixedTCOs() && me->button() == Qt::LeftButton )
 	{
-		if( me->modifiers() & Qt::ControlModifier )
+		if( m_trackView->trackContainerView()->knifeMode() && !m_tco->getAutoResize() )
+		{
+			SampleTCO * sTco = dynamic_cast<SampleTCO*>( m_tco );
+
+			if( me->x() < RESIZE_GRIP_WIDTH && sTco )
+			{
+				m_action = ResizeLeft;
+				setCursor( Qt::SizeHorCursor );
+			}
+			else if( me->x() >= width() - RESIZE_GRIP_WIDTH )
+			{
+				m_action = Resize;
+				setCursor( Qt::SizeHorCursor );
+			}
+			else if (sTco)
```

Comment:
Can we add a virtual method to check if a TCO supports splitting? Eventually for left resizing as well, but it's out of scope.